lbrynet/core/Session.py

```diff
@@ -119,6 +121,7 @@ def __init__(self, blob_data_payment_rate, db_dir=None,
         self.blob_tracker_class = blob_tracker_class or BlobAvailabilityTracker
 
         self.peer_port = peer_port
+        self.external_peer_port = peer_port
 
```

Comment:
dht.node class be initialized with self.external_peer_port instead of self.peer_port for its peerPort argument.

Reply:
Is this something you want me to change in this PR? I was planning to actually "use" the external port fields in a next PR.

Reply:
what was going to be in this PR? Other than this change?

Reply:
Yes, right, but this is what is already happening right now in the code. It is not introduced by this PR.
But I can append some more commits. I just wanted to push some work out for review and this is happening already :)